[2022-06-05 22:55:01,891] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-06T01:54:02.888035+00:00 [queued]>
[2022-06-05 22:55:01,911] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-06T01:54:02.888035+00:00 [queued]>
[2022-06-05 22:55:01,911] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 22:55:01,911] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 22:55:01,912] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 22:55:01,935] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): populate_table> on 2022-06-06 01:54:02.888035+00:00
[2022-06-05 22:55:01,946] {standard_task_runner.py:52} INFO - Started process 41805 to run task
[2022-06-05 22:55:01,952] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'populate_table', 'scheduled__2022-06-06T01:54:02.888035+00:00', '--job-id', '1184', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp5ejmvg43', '--error-file', '/tmp/tmpy8zc7i9w']
[2022-06-05 22:55:01,953] {standard_task_runner.py:80} INFO - Job 1184: Subtask populate_table
[2022-06-05 22:55:02,035] {task_command.py:370} INFO - Running <TaskInstance: census.populate_table scheduled__2022-06-06T01:54:02.888035+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 22:55:02,143] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=populate_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-06T01:54:02.888035+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-06T01:54:02.888035+00:00
[2022-06-05 22:55:02,911] {python.py:173} INFO - Done. Returned value was: 1630
[2022-06-05 22:55:02,935] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=populate_table, execution_date=20220606T015402, start_date=20220606T015501, end_date=20220606T015502
[2022-06-05 22:55:03,006] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 22:55:03,069] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
